rudder-web/src/main/resources/demo-rudder-users.xml

```diff
@@ -64,7 +64,7 @@
 
   Please take a look at Rudder's documentation for additional details, either from the
   application itself (upper right corner) or on
-  http://www.rudder-project.org/rudder-doc-latest/rudder-doc.html#user-management.
+  https://docs.rudder.io/reference/current/administration/users.html
```

Comment:
this is a 404

Reply:
indeed.... this redirection looks broken now